packages/node-red-s1seven-api/lib/tokens/get-token.js

```diff
@@ -30,8 +31,22 @@ module.exports = function (RED) {
 
     /** @type {import('../utils/setters')} */
     const setters = scope.resolve('setters');
+    /** @type {import('../utils/getters')} */
+    const getters = scope.resolve('getters');
 
-    node.on('msg', async (msg, send, done) => {
+    node.on('msg', async (msg, sendCb, done) => {
+      /**
+       * @param {[NodeMessage, null] | [null, NodeMessage]} inputArr
+       */
+      function send(inputArr) {
+        const oldAccessToken = getters.getAccessToken();
+        const newAccessToken = inputArr[0]?.payload.accessToken;
+        const eventObj = {
+          event: 'has-changed',
+          value: Boolean(newAccessToken) && oldAccessToken !== newAccessToken,
+        };
+        sendCb([...inputArr, eventObj]);
+      }
```

Comment:
i feel like it would be simpler to just emit another message with `send` in case of success => [here](https://github.com/s1seven/node-red-monorepo/blob/a685208f83f1dba45d1854e5cc21e2b5c4e0e587/packages/node-red-s1seven-api/lib/tokens/get-token.js#L74)

like 
```js
send([null, null, {
   event: 'has-changed',
   value: node.getters.getAccessToken() !== data.accessToken,
}])
```

Reply:
That was my original solution 😄